### PR TITLE
Update to 3.2.1 and latest gems

### PR DESCRIPTION
**Pull request by iHiD**
This stops the warning I get on my new mac.

**Note: This needs the Dockerfile updating to 3.2.1. before we deploy this!**

### CODE REVIEW

File: app/commands/donations/subscription/cancel.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Donations::Subscription::Cancel
 
   def call
     begin
-      Stripe::Subscription.delete(subscription.stripe_id)
+      Stripe::Subscription.cancel(subscription.stripe_id)
```
Comment (ErikSchierboom) [inline app/commands/donations/subscription/cancel.rb:10]: See https://github.com/stripe/stripe-ruby/blob/master/CHANGELOG.md#%EF%B8%8F-removed

File: app/commands/track/determine_concept_map_layout.rb [hunk 2]
```diff
@@ -92,9 +92,7 @@ def initialize(user_track)
     end
 
     memoize
-    def empty?
-      nodes.empty?
-    end
+    delegate :empty?, to: :nodes
```
Comment (ErikSchierboom) [inline app/commands/track/determine_concept_map_layout.rb]: Would this (memoize) still work when using `delegate`?

File: config/initializers/devise.rb [hunk 3]
```diff
@@ -1,17 +1,3 @@
-class TurboFailureApp < Devise::FailureApp
```
Comment (ErikSchierboom) [inline config/initializers/devise.rb:1]: The changes in this file are taken from https://github.com/heartcombo/devise/issues/5560

File: app/commands/track/determine_concept_map_layout.rb (hunk 2 above)
Comment (iHiD) [reply app/commands/track/determine_concept_map_layout.rb]: No.

**Pull request merged**